src/test/scala/org/virtuslab/akkasaferserializer/TestBorerAkkaSerializer.scala

```diff
@@ -0,0 +1,13 @@
+package org.virtuslab.akkasaferserializer
+
+class TestBorerAkkaSerializer extends CborAkkaSerializer[BorerSerializable] {
+  import Codecs._
+
+
+  override def identifier: Int = 19923
```

Comment:
Pls remind me, is this just a random value? or what's its origin?

Reply:
From Akka documentation:
```
/**
 * Completely unique value to identify this implementation of Serializer, used to optimize network traffic.
 * Values from 0 to 40 are reserved for Akka internal usage.
 */
def identifier: Int
```

Reply:
It is random in a sense. I didn't think much when I was hitting the keyboard.